src/components/base/SchoolSearch.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { ChangeEvent, useRef, useState } from 'react';
+import { schools } from '@/mock/schools';
+import styled from 'styled-components';
+import { DeleteIcon, SearchIcon } from '@/assets/img';
+
+const SchoolSearch = () => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const [selectedSchools, setSelectedSchools] = useState<string[]>([]);
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    const { value } = e.target;
+    if (!schools.map(({ name }: { name: string }) => name).includes(value)) return;
+    setSelectedSchools((prev) => [...prev, value]);
+    inputRef.current!.value = '';
```

Comment:
여기 lint 경고떠서 번거롭겠지만 수정하면 좋을 것 같아요!
```suggestion
    if (inputRef.current) inputRef.current.value = '';
```

Reply:
base는 기초 컴포넌트 들어가는 곳이라 components/domain에 들어가는게 더 좋겠네요

Reply:
수정했습니다! 감사합니다~